### PR TITLE
R4 Final Review

### PR DESCRIPTION
**Pull request by amsmith-pro**
https://www.arrowair.com/docs/contributing/guides/release-checklist

### CODE REVIEW

**Comment by owlot** [comment]
## Documents
- [x] ~~ICD~~ (no changes)
- [x] SDD
  - [x] Description of grpc & rest handler implementations
  - [x] Current sequence diagrams where applicable
  - [x] Current relevant document links
- [x] ~~CONOPS~~ (no changes)
- [x] README (see svc-cargo for example)
  - [x] Badges
  - [x] Accurate installation instructions
  - [x] Current links to other documents
- [x] Any instance of :construction: should genuinely be under construction

## Code (not applicable to examples/ folder)
- [x] No unhandled unwrap()s
- [x] Accurate (!) code comments on all functions
- [x] No `# Arguments` and `# Returns` in function comments
- [x] Function arguments checked where possible
- [x] Sufficient logger (info, warn, and debug) messages
- [x] Sensible Types (unsigned vs. signed)
- [x] No `#[allow(dead_code)]` without good reason
- [x] End all error, info, and debug messages with punctuation (periods, etc.)
- [x] No FIXMEs or TODOs without a projected release (`TODO R4: etc.`)

## REST API
- [x] Return codes accurately documented with openapi
- [x] Return codes make sense
- [x] Sufficiently descriptive error messages
- [x] Do NOT reflect a user's bad input back at them in replies.
    - i.e. "Warning: '{}' is not a valid entry." should instead be "Warning: invalid entry."
    - Use debug messages to see the invalid argument.

## Cargo.toml(s)
- [x] Package versions correct
- [x] No minor versions specified
- [x] Links to crates.io package, not git branch
- [x] Must have `license-file = "LICENSE"` under `[package]`

File: docs/sdd.md
Comment (owlot) [inline docs/sdd.md:1]: This document is describing an `atc/ack/flights` endpoint which should probably be changed to `/atc/acknowledge` as per the source code API definition

File: docs/sdd.md [hunk 1]
```diff
@@ -81,31 +81,27 @@ sequenceDiagram
     participant service as svc-atc
     participant storage as svc-storage
```
Comment (owlot) [inline docs/sdd.md]: svc-storage doesn't seem needed in this scenario

Comment (amsmith-pro) [reply docs/sdd.md]: https://github.com/aetheric-oss/svc-atc/pull/11/commits/0e09d8d5822f11cd187bcdd0dae377202a5b2423

File: docs/sdd.md
Comment (amsmith-pro) [reply docs/sdd.md:1]: https://github.com/aetheric-oss/svc-atc/pull/11/commits/0e09d8d5822f11cd187bcdd0dae377202a5b2423

**Review by owlot — APPROVED** [review]

**Pull request merged**